utils/src/shutdown.rs

```diff
@@ -188,6 +196,24 @@ impl Listener {
     }
 }
 
+/// sync follower token, used to track the sync follower tasks.
+#[derive(Debug)]
+pub struct SyncFollowerToken {
```

Comment:
What's the difference between `SyncFollowerToken` and `Trigger`? They looks very similar.

Reply:
`SyncFollowerToken` is only used to track the number of `synf_follower_task`, just rely on its drop method. `Trigger` Contains additional logic related to shutdown, and this will be refactor in `TaskManager`